### DISCUSSION TITLE
Help with Unstructured Grid Specific Array Plotting

### DISCUSSION THREAD

**Discussion by chcarra** [pyvista/pyvista#4185] [Q&A] [answered]
I have an unstructured grid with N Arrays. Whenever I specify grid.plot(), I only get the first array, but I am interested in plotting specific arrays by passing in the array name (or key) somehow.

**Comment by banesullivan** [comment, marked as answer]
Try the `scalars='name of array'` keyword argument

**Reply by chcarra** [reply]
That works! Thank you so much!
